src/Tyler.jl

```diff
@@ -134,12 +131,12 @@ function Map(extent, extent_crs=wgs84;
         fetched_tiles,
         max_parallel_downloads, OrderedSet{Tile}(),
         tiles_being_added, downloaded_tiles,
-        display_task, download_task, screen,
+        display_task, download_task, 
         depth, halo, scale
     )
     tyler.zoom[] = get_zoom(tyler, extent)
     download_task[] = @async begin
-        while isopen(screen)
+        while true
```

Comment:
Does this mean the task runs forever now?

Reply:
That's the big question, I am not quite sure about that, but had no real alternative without the screen in mind.

Reply:
My guess is this task will just keep running every 0.01 seconds until you kill the session, unless it hits another error from the Makie screen not being there anymore (but I think it wont?).

We can't really do that... we need to find another way of breaking the loop.

Reply:
Maybe try prining to stdout in the task and see what happens